tests/valgrind.suppression

```diff
@@ -1,131 +1,87 @@
 {
    <insert_a_suppression_name_here>
    Memcheck:Leak
-   match-leak-kinds: reachable
-   fun:_Znwm
-   obj:/usr/lib/libbotan-1.10.so.1.12
-   obj:/usr/lib/libbotan-1.10.so.1.12
-   fun:_ZN5Botan17Algorithm_Factory13prototype_macERKNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEES8_
-   fun:_ZN5Botan17Algorithm_Factory12providers_ofERKNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEE
-   fun:_ZN5Botan13algorithm_katERKNS_9SCAN_NameERKSt3mapINSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEES9_St4lessIS9_ESaISt4pairIKS9_S9_EEERNS_17Algorithm_FactoryE
-   obj:/usr/lib/libbotan-1.10.so.1.12
-   fun:_ZN5Botan26confirm_startup_self_testsERNS_17Algorithm_FactoryE
-   fun:_ZN5Botan13Library_State10initializeEb
-   fun:_ZN5Botan18LibraryInitializer10initializeERKNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEE
-   obj:*
-   obj:*
-   fun:elektraPluginOpen
-   fun:test_crypto_operations
-   fun:main
+   match-leak-kinds: all
+   ...
+   obj:*libbotan*
+   ...
 }
 {
    <insert_a_suppression_name_here>
    Memcheck:Leak
-   match-leak-kinds: reachable
-   fun:malloc
-   obj:*libgcrypt.so.20*
-   obj:*libgcrypt.so.20*
-   obj:*libgcrypt.so.20*
-   obj:*libgcrypt.so.20*
-   fun:elektraCryptoGcryInit
-   fun:test_init
-   fun:main
+   match-leak-kinds: all
+   ...
+   fun:_ZN5Botan18LibraryInitializer10initializeERKSs
+   ...
 }
 {
    <insert_a_suppression_name_here>
-   Memcheck:Addr4
-   obj:*
-   obj:*
-   obj:*
-   obj:*
-   obj:*
+   Memcheck:Leak
+   match-leak-kinds: all
+   ...
+   obj:*libgcrypt*
    ...
-   fun:JNI_CreateJavaVM
-   fun:elektraJniOpen
-   fun:elektraPluginOpen
 }
 {
    <insert_a_suppression_name_here>
    Memcheck:Cond
-   fun:is_overlap
-   fun:memcpy@@GLIBC_2.14
-   fun:keySetRaw
+   ...
    fun:elektraCryptoGcryDecrypt
```

Comment:
I tried several things in `CryptoGcryDecrypt` and came to the conclusion that `gcry_cipher_decrypt` is causing the conditional jump. The valgrind message comes in different variations, including one which has no `keySetRaw` as a frame.